chainntnfs/btcdnotify/btcd.go

```diff
@@ -27,6 +27,31 @@ const (
 	reorgSafetyLimit = 100
 )
 
+// txConfStatus denotes the status of a transaction's lookup.
+type txConfStatus uint8
+
+const (
```

Comment:
Note: as discussed, would be nice to later make these shared between all backends, and then include them in `SpendEvents` etc